File: PapelTesoura/app/src/main/java/com/example/papeltesoura/MainActivity.java
package com.example.papeltesoura;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void selecionarPedra(View view){
        verificarResultado("pedra");
    }

    public void selecionarPapel(View view){
        verificarResultado("papel");
    }

    public void selecionarTesoura(View view){
        verificarResultado("tesoura");
        resultadoMaquina();
    }

    private void resultadoMaquina(View view){
        Maquina maquina = new Maquina();
        ImageView imagemApp = new 

    }

    private void verificarResultado( String escolhaUsuario){
        System.out.println("usuario selecionou: " + escolhaUsuario);
    }


}

File: testeToast/app/src/main/java/com/example/testetoast/MainActivity.kt
package com.example.testetoast

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    lateinit var btnClique: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnClique = findViewById(R.id.btn_clique)
        btnClique.setOnClickListener {
            Toast.makeText(this, "Parabens, você clicou", Toast.LENGTH_SHORT).show()
        }

    }
}